x-pack/plugins/fleet/server/routes/package_policy/index.ts

```diff
@@ -61,7 +64,22 @@ export const registerRoutes = (router: FleetAuthzRouter) => {
         integrations: { writeIntegrationPolicies: true },
       },
     },
-    createPackagePolicyHandler
+    authzHandlerWrapper(
+      {
+        fleetAuthz: {
+          integrations: { writeIntegrationPolicies: true },
+        },
+        fleetPackageAuthz: {
+          managePackagePolicy: true,
+          getPackageFromRequest: (req: any) => {
+            return (req as CreatePackagePolicyRequest).body.package?.name;
```

Comment:
authz config specifies `getPackageFromRequest` function that tells how to take package from request object